### PR TITLE
Rerun tutorials with SimPEG v0.22

### PR DESCRIPTION
**Pull request by santisoler**
Bump simpeg version in `environment.yml` to v0.22 and rerun all notebooks with SimPEG v0.22. Replace `SimPEG` for the new `simpeg` throughout notebooks and Markdown files. Add admonitions mentioning the ability to run the potential field simulations using Choclo. Fix typo in `declination` argument for magnetics' `UniformBackgroundField`. Minor improvement in instruction to run all notebooks in `README.md`.

### CODE REVIEW

File: notebooks/03-gravity/fwd_gravity_anomaly_3d.ipynb [hunk 1]
```diff
@@ -609,6 +611,7 @@
     "    rhoMap=model_map,\n",
     "    ind_active=active_cells,\n",
     "    store_sensitivities=\"forward_only\",\n",
+    "    engine=\"choclo\",\n",
```
Comment (dccowan) [inline notebooks/03-gravity/fwd_gravity_anomaly_3d.ipynb:614]: For introductory tutorials like this, I do not want to use functionality that comes from soft dependencies. This notebook should run with basic simpeg. I think we should:
* expose the `engine` argument but set it to the default value
* reword the markdown talking about the choclo engine accordingly.

Comment (santisoler) [reply notebooks/03-gravity/fwd_gravity_anomaly_3d.ipynb:614]: Makes sense! I'll apply these changes then.

**Comment by santisoler** [comment]
I need to rerun the notebooks now.

File: notebooks/03-gravity/fwd_gravity_anomaly_3d.ipynb (hunk 1 above)
Comment (dccowan) [reply notebooks/03-gravity/fwd_gravity_anomaly_3d.ipynb]: Looks great!

**Pull request merged**